android/app/src/main/java/app/organicmaps/search/SearchResult.java

```diff
@@ -141,4 +142,37 @@ public Spannable getFormattedTitle(@NonNull Context context)
 
     return builder;
   }
+
+  public String getAddress(@NonNull Context context)
+  {
+    return description.region;
+  }
+
+  public Spannable getFormattedAddress(@NonNull Context context)
+  {
+      final String address = getAddress(context);
+      final String addr = address.toLowerCase();
+      String title = getTitle(context);
+      final SpannableStringBuilder builder = new SpannableStringBuilder(address);
+
+      if (highlightRanges != null)
+      {
+        final int size = highlightRanges.length / 2;
+        int index = 0;
+
+        for (int i = 0; i < size; i++)
+        {
+          final int start = highlightRanges[index++];
+          final int len = highlightRanges[index++];
```

Comment:
Is it possible to avoid using index variable?

Reply:
We have locally defined index variable to iterate over the array so we need that index.

Please check the function getFormattedTitle which is used to highlight titles, I took reference from there.

There also index variable was used so i used it in similar fashion.

Reply:
That reference function is not the best possible example )

Can `i` be used instead of `index`?

Reply:
Yes, "i" can be used.